src/test/java/uk/gov/hmcts/reform/civil/service/flowstate/StateFlowEngineTest.java

```diff
@@ -1059,12 +1123,12 @@ void shouldReturnDivergentResponseAndGoOffline_1v2Scenario_whenFirstRespondentSu
                     .isNotNull()
                     .isEqualTo(DIVERGENT_RESPOND_GO_OFFLINE.fullName());
                 assertThat(stateFlow.getStateHistory())
-                    .hasSize(10)
+                    .hasSize(9)
                     .extracting(State::getName)
                     .containsExactly(
                         DRAFT.fullName(), CLAIM_SUBMITTED.fullName(), CLAIM_ISSUED_PAYMENT_SUCCESSFUL.fullName(),
                         PENDING_CLAIM_ISSUED.fullName(), CLAIM_ISSUED.fullName(), CLAIM_NOTIFIED.fullName(),
-                        CLAIM_DETAILS_NOTIFIED.fullName(), NOTIFICATION_ACKNOWLEDGED.fullName(),
+                        CLAIM_DETAILS_NOTIFIED.fullName(),
```

Comment:
there should be a flow state NOTIFICATION_ACKNOWLEDGED.fullName() here as well as both of them have acknowledged the claim. Please update the test

Reply:
In this case, respondent 2 hasn't acknowledged, only responded. So flow state won't be containing NOTIFICATION_ACKNOWLEDGED state.

Reply:
the title of the tests says **_whenBothPartiesAcknowledgedAndSubmitFullDefenceResponses** please update the title of the test in that case and add a new test where both parties have acknowledged the claim so that we verify the state flow has the state NOTIFICATION_ACKNOWLEDGED.fullName()